libs/libsigar/Makefile

```diff
@@ -25,7 +25,6 @@ define Package/libsigar
 	TITLE:=System Information Gatherer And Reporter
 	URL:=https://github.com/boundary/sigar
 	DEPENDS:=+libtirpc
-	MAINTAINER:=Amol Bhave <ambhave@fb.com>
```

Comment:
I've still not seen _anything_ that says this is wrong, you're jsut makign gratuitous changes like this.